apps/etterlatte-vilkaarsvurdering/src/main/kotlin/vilkaarsvurdering/VilkaarsvurderingService.kt

```diff
@@ -156,7 +162,42 @@ class VilkaarsvurderingService(
                         behandling.sak,
                         brukerTokenInfo
                     )
-                    kopierVilkaarsvurdering(behandlingId, sisteIverksatteBehandling.id, brukerTokenInfo)
+
+                    val forrigeVilkaarsvurdering = vilkaarsvurderingRepository.hent(sisteIverksatteBehandling.id)
+                        ?: throw NullPointerException(
+                            "Fant ikke vilkårsvurdering fra behandling ${sisteIverksatteBehandling.id}"
+                        )
+
+                    val vilkaar = hentVilkaarForRevurdering(
```

Comment:
Hent vilkår for revudering hvis type er førstegangsbehandling...? Skurrer litt for meg 🤔